### PR TITLE
Update script with AMBER and CHARMM 

### LINKED ISSUE #123: Add forcefield options to the scripts
MDPOW script doesn't allow users to choose forcefileld with yml file.
OPLS-AA is the only choice when using the script with yml input.

### PR DESCRIPTION
**Pull request by VOD555**
Fix #123 
-add option for forcefield
-make wetoctanol toplogy file work with gromacs 2018 or higher version
-add mdp files for charmm

### CODE REVIEW

File: mdpow/templates/bar_charmm.mdp [hunk 1]
```diff
@@ -0,0 +1,140 @@
+; Gromacs mdp file
+; Template for MD Hydration FEP with OPLS-AA (part of POW)
+;
+; grompp -pp -c ../../../MD_POSRES/md.gro -p ../../../top/system.top -f fep_opls.mdp -o md.tpr
+
+
+include                  = 
+define                   = 
+
+integrator               = sd
```
Comment (orbeckst) [inline mdpow/templates/bar_charmm.mdp:10]: Have we always been doing SD? Didn't we use v-rescale recently?

File: mdpow/templates/bar_charmm.mdp [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+pme_order                = 6
+ewald_rtol               = 1e-06
+ewald_geometry           = 3d
+epsilon_surface          = 0
+optimize_fft             = yes
+
+; OPTIONS FOR WEAK COUPLING ALGORITHMS
+; Temperature coupling (use Langevin for free energy calculations, see
+; Mobley and Shirt papers)
+Tcoupl                   = no
```
Comment (orbeckst) [inline mdpow/templates/bar_charmm.mdp:77]: This might be outdated... didn't we do v-rescale for SAMPL6?

**Comment by orbeckst** [comment]
@VOD555 we did use Langevin (SD) for SAMPL6, so your charmm mdp is fine.
@iorga are we using Langevin for T-coupling for SAMPL7 as well?

**Comment by iorga** [comment]
> @iorga are we using Langevin for T-coupling for SAMPL7 as well?

Yes, we are using `mdpow/templates/bar_opls.mdp` which has the SD integrator, so Langevin for T-coupling.

The only FEP sims that were run for SAMPL7 are the long simulations for SM30 and SM45 with OPLS-AA, so we can change now these options if needed.

**Comment by orbeckst** [comment]
> The only FEP sims that were run for SAMPL7 are the long simulations for SM30 and SM45 with OPLS-AA, so we can change now these options if needed.

Let's keep it the same way as before – one thing less to double check.

**Comment by iorga** [comment]
@VOD555 I just realized that in the name of this PR is also AMBER, but only CHARMM was implemented.

On the other hand, I just checked and the default `runinput.yml` file (generated by `mdpow-get-runinput`) is not updated with the forcefield and prm new options.

Could you please make these modification? Thanks.

**Comment by VOD555** [comment]
@iorga As amber doesn't require some specific parameters like CHARMM, we can just use the mdp files for OPLS-AA, which are the default mdp file. 

The options for forcefield and prm can be found on my local test. It's inside `setup` dictionary(the same as https://github.com/Becksteinlab/MDPOW/blob/develop/mdpow/templates/runinput.yml).

**Comment by iorga** [comment]
@VOD555 Many thanks. My comment was not about the .mdp file, but about the .top file, which in OPLS-AA makes mention to the oplsaa.ff folder. But I checked now and the correct folders are used in the .top file for CHARMM and AMBER.

Yes, indeed, these options are in the `runinput.yml` file from the templates folder. But the regular user would look at the one generated by the `mdpow-get-runinput` script, which was created for this purpose and should be updated at some point. I will try and see if I can do this update.

**Comment by orbeckst** [comment]
Good point, thanks for catching the mdpow-get-runinput script – I only saw the runinput.yml and was happy.

**Comment by iorga** [comment]
I finally found that in my installation the template `runinput.yml` file was not being updated. Everything is fine now, the `mdpow-get-runinput` script indeed produces the `runinput.yml` file with the updated options.

**Pull request merged**